packages/plugins/experimental/plugin-transcription/src/components/TranscriptContainer.tsx

```diff
@@ -129,12 +129,32 @@ const summarizeTranscript = async (
 
 // TODO(dmaretskyi): Add example to set consistent structure for the summary.
 const SUMMARIZE_PROMPT = `
-  Create a summary of the transcript provided.
-  Reference specific people by name when possible.
-  Format the summary as a list of key points and takeaways.
-  Include action items in the summary.
-  Use markdown formatting for headings and bullet points.
-  Format the summary as a markdown document without extra comments like "Here is the summary of the transcript:".
+  <GOAL>
+  Create a markdown summary of the transcript provided.
+  </GOAL>
+
+  <FORMATTING>
+  - Format the summary as a markdown document without extra comments like "Here is the summary of the transcript:".
+  - Use markdown formatting for headings and bullet points.
+  - Format the summary as a list of key points and takeaways.
+  - All names of people should be in bold.
+  </FORMATTING>
+
+  <NOTE_TAKING>
+  - Correlate items in the summary with the person of origin to build a coherent narrative.
+  - Include short quotes verbatim where appropriate. Especially when concerned with design decisions and problem descriptions.
+  </NOTE_TAKING>
+
+  <TASKS>
+  At the end of the summary include tasks.
+  Extract only the tasks that are:
+  - Directly actionable
+  - Clearly assigned to a person or team (or can easily be inferred)
+  - Strongly implied by the conversation and/or user note (no speculative tasks)
+  - Specific enough that someone reading them would know exactly what to do next
```

Comment:
require markdown formatting -- i.e., `- [ ] xxx.`
this might be referred to as GFM (github flavored markdown?)